gradle.properties

```diff
@@ -15,18 +15,15 @@ org.gradle.parallel.threads=8
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
-org.gradle.java.home=/Applications/Android Studio.app/Contents/jre/jdk/Contents/Home
 # ---------------------------------- MiniApp SDK flags ------------------------------------
 # Artifactory information needed.
 group=com.rakuten.tech.mobile.miniapp
 name=miniappplatform
-#"-" indicates aar file will be uploaded to snapshot.
-version=0.0.1-SNAPSHOT
-versionCode=1
 url=https://rakutentech.github.io/android-miniapps
 # MiniApp SDK info for Android.
 application_id=com.rakuten.mobile.miniappsplatform
```

Comment:
Do you know why `application_id` is being set here? I think we don't need it.

Reply:
It's something coming over from the previous structure. It is not defined in the `build.gradle` and the previous intent seems to be setting it up from the gradle.properties instead.
I also think, we can remove this going forward, along with the changes for publishing.